### PR TITLE
Move Exception into getConfiguration method

### PR DESCRIPTION
**Pull request by ker0x**
Exception on missing configuration was only thrown when using a command. This fix that by moving Exception directly into `getConfiguration()` method which is also called in `WebUIController`

### CODE REVIEW

File: Tests/Unit/Service/ConfigurationManagerTest.php [hunk 1]
```diff
@@ -46,13 +46,16 @@ public function testGetConfigurationDefault(): void
...
+        $this->expectExceptionMessage('No configuration found for "missing"');
+
         $manager = new ConfigurationManager();
         $correctConfiguration = $this->createConfiguration(['name' => 'correct']);
 
         $manager->addConfiguration('bar', $this->createConfiguration());
         $manager->addConfiguration('default', $correctConfiguration);
 
-        $this->assertNull($manager->getConfiguration('missing'));
+        $config = $manager->getConfiguration('missing');
```
Comment (odolbeau) [inline Tests/Unit/Service/ConfigurationManagerTest.php]: Unused variable

File: Tests/Unit/Service/ConfigurationManagerTest.php [hunk 2]
```diff
@@ -46,13 +46,16 @@ public function testGetConfigurationDefault(): void
 
     public function testGetConfigurationMissing(): void
     {
+        $this->expectException(\InvalidArgumentException::class);
```
Comment (odolbeau) [inline Tests/Unit/Service/ConfigurationManagerTest.php]: Could you move those line just before `$manager->getConfiguration('missing');` as it's where you expect the test to fail?

File: Tests/Unit/Service/ConfigurationManagerTest.php (hunk 1 above)
Comment (ker0x) [reply Tests/Unit/Service/ConfigurationManagerTest.php]: Fixed 👍

File: Tests/Unit/Service/ConfigurationManagerTest.php (hunk 2 above)
Comment (ker0x) [reply Tests/Unit/Service/ConfigurationManagerTest.php]: Also fixed !

**Comment by odolbeau** [comment]
Perfect! :+1:

**Pull request merged**